packages/wonder-blocks-dropdown/components/action-menu.js

```diff
@@ -163,57 +177,65 @@ export default class ActionMenu extends React.Component<MenuProps, State> {
             // Item was newly selected
             onChange([...selectedValues, selectedValue]);
         }
-    }
+        this.handleItemSelected();
+    };
 
-    getMenuItems(): Array<
-        React.Element<
-            typeof ActionItem | typeof OptionItem | typeof SeparatorItem,
-        >,
-    > {
+    getMenuItems(): Array<DropdownItem> {
         const {children, selectedValues} = this.props;
         const containsOptionItems = Array.isArray(selectedValues);
 
-        return React.Children.map(children, (item, index) => {
+        return React.Children.map(children, (item) => {
             if (item.type === ActionItem) {
-                return React.cloneElement(item, {
-                    indent: containsOptionItems,
-                });
+                const {disabled, onClick} = item.props;
+                return {
+                    component: item,
+                    focusable: !disabled,
+                    populatedProps: {
```

Comment:
This might be a little cleaner if we did `cloneElement` here and passed these additional props here.  The logic around `focusable` could be consolidated in `Dropdown` by checking `item.props.disabled` and `item.type` against each of the item types.

Reply:
That's what I was originally doing, but I wasn't able to add the `ref` to `children` of `Dropdown` without recloning each element, and I thought having two separate cloning processes was bad.

Reply:
That's what I was originally doing, but I wasn't able to add the `ref` to  each child in `children` of `Dropdown` without recloning each element, and I thought having two separate cloning processes was bad.